Mage.Sets/src/mage/cards/r/RivazOfTheClaw.java

```diff
@@ -0,0 +1,76 @@
+package mage.cards.r;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.common.CastFromGraveyardOnceStaticAbility;
+import mage.abilities.common.DiesSourceTriggeredAbility;
+import mage.abilities.common.SpellCastControllerTriggeredAbility;
+import mage.abilities.effects.common.ExileSourceEffect;
+import mage.abilities.effects.common.continuous.GainAbilityTargetEffect;
+import mage.abilities.mana.ConditionalAnyColorManaAbility;
+import mage.abilities.mana.conditional.ConditionalSpellManaBuilder;
+import mage.constants.*;
+import mage.abilities.keyword.MenaceAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.filter.common.FilterCreatureCard;
+import mage.filter.common.FilterCreatureSpell;
+import mage.filter.predicate.card.CastFromZonePredicate;
+
+/**
+ *
+ * @author weirddan455
+ */
+public final class RivazOfTheClaw extends CardImpl {
+
+    private static final FilterCreatureSpell filter = new FilterCreatureSpell("Dragon creature spells");
+    private static final FilterCreatureCard filter2 = new FilterCreatureCard();
+    private static final FilterCreatureSpell filter3 = new FilterCreatureSpell("a Dragon creature spell from your graveyard");
```

Comment:
Could you please change their names to reflect which ability they're for. 

I can imagine these would be so easy to mix up in any future changes or refactor